conf/metadata.php

```diff
@@ -5,6 +5,7 @@
  * @author Wolfgang Gassler <wolfgang@gassler.org>
  */
 
+$meta['initRepo'] = array('onoff');
```

Comment:
Should we _really_ have a case where the repo can be not initialized? IMOH, we have 2 cases. Either it is a repo, and it should behave as a one (commit on change) or it is not, and it should behave as a pure file tree (no commit on change). It is your job as a sysadmin, not to put files out of a git repository.
What do you think? Is it a sane thought?

Reply:
I'll check how it behave in your branch and propose a solution if it is still the case.